internal/cmd/asyncapi/command_export.go

```diff
@@ -306,13 +319,53 @@ func (c *command) getBindings(clusterId, topicName string) (*bindings, error) {
 	if partitionsResp.Data != nil {
 		numPartitions = int32(len(partitionsResp.Data))
 	}
-	configsMap := make(map[string]string)
+	customConfigMap := make(map[string]string)
+	topicConfigMap := make(map[string]any)
+
+	// Determine whether the given config value can be put into the AsyncAPI Kakfa bindings or put into our custom struct for extra configs
+	topicConfigOptions := types.NewSet(
+		"cleanup.policy",
+		"delete.retention.ms",
+		"max.message.bytes",
+		"retention.bytes",
+		"retention.ms",
+	)
 	for _, config := range configs.Data {
-		configsMap[config.GetName()] = config.GetValue()
+		if topicConfigOptions.Contains(config.GetName()) {
+			switch config.GetName() {
+			case "cleanup.policy":
+				topicConfigMap[config.GetName()] = strings.Split(config.GetValue(), ",")
+			case "max.message.bytes":
+				topicConfigMap[config.GetName()], err = strconv.ParseInt(config.GetValue(), 10, 32)
+				if err != nil {
+					return nil, err
+				}
+			default:
```

Comment:
Does this encompass "delete.retention.ms", "retention.bytes", and "retention.ms"? Can we write all of those to be more explicit?

That'll let us use the `default` block to add to the custom config map, and then we won't need to use a `set` anymore!